pkg/datastore/database.go

```diff
@@ -216,16 +215,28 @@ func (db *relationalDb) Find(ctx context.Context, record Record) error {
 	return db.FindInTable(ctx, GetTableName(record), record)
 }
 
+func (db *relationalDb) FindSoftDeleted(ctx context.Context, record Record) error {
+	return db.FindSoftDeletedInTable(ctx, GetTableName(record), record)
+}
+
 func rollbackTx(tx *gorm.DB, db *relationalDb) {
```

Comment:
move to top

Reply:
please check latest commit. Deleted unused functions

Reply:
nit: just have all related functions like Find/FindSoftDeleted/FindInTable all close to each other and rollbackTx is common can be moved closer to first method definition.